include/ocpp/v201/ocpp_types.hpp

```diff
@@ -233,12 +480,59 @@ void from_json(const json& j, ChargingProfileCriterion& k);
 /// \returns an output stream with the ChargingProfileCriterion written to
 std::ostream& operator<<(std::ostream& os, const ChargingProfileCriterion& k);
 
+struct V2XFreqWattPoint {
+    float frequency;
+    float power;
+    std::optional<CustomData> customData;
+};
+/// \brief Conversion from a given V2XFreqWattPoint \p k to a given json object \p j
+void to_json(json& j, const V2XFreqWattPoint& k);
+
+/// \brief Conversion from a given json object \p j to a given V2XFreqWattPoint \p k
+void from_json(const json& j, V2XFreqWattPoint& k);
+
+// \brief Writes the string representation of the given V2XFreqWattPoint \p k to the given output stream \p os
+/// \returns an output stream with the V2XFreqWattPoint written to
+std::ostream& operator<<(std::ostream& os, const V2XFreqWattPoint& k);
+
+struct V2XSignalWattPoint {
+    int32_t signal;
+    float power;
+    std::optional<CustomData> customData;
+};
+/// \brief Conversion from a given V2XSignalWattPoint \p k to a given json object \p j
+void to_json(json& j, const V2XSignalWattPoint& k);
+
+/// \brief Conversion from a given json object \p j to a given V2XSignalWattPoint \p k
+void from_json(const json& j, V2XSignalWattPoint& k);
+
+// \brief Writes the string representation of the given V2XSignalWattPoint \p k to the given output stream \p os
+/// \returns an output stream with the V2XSignalWattPoint written to
+std::ostream& operator<<(std::ostream& os, const V2XSignalWattPoint& k);
+
 struct ChargingSchedulePeriod {
     int32_t startPeriod;
-    float limit;
     std::optional<CustomData> customData;
+    std::optional<float> limit;
```

Comment:
TODO: find out implications for backwards compatibility with 2.0.1
This was required in 2.0.1 and is optional now in 2.1